subscription/downgrade.md

```diff
@@ -7,36 +7,30 @@ redirect_from:
 - /docker-hub/billing/downgrade/
 ---
 
-You can downgrade your Docker subscription anytime. When you downgrade your subscription, changes are applied immediately, and you will no longer have access to the prior services.
+You can downgrade your Docker subscription at anytime before the renewal date. The unused portion of the subscription isn't refundable.
```

Comment:
**suggestion:** Docker neither refund (aka send money back to a debit/credit card) nor credit (add money to a billing account for future usage).

```suggestion
You can downgrade your Docker subscription at anytime before the renewal date. The unused portion of the subscription isn't refundable nor creditable.
```